config/default/manager_metrics_service.yaml

```diff
@@ -10,8 +10,8 @@ metadata:
 spec:
   ports:
   - name: https
-    port: 8080
+    port: 8443
     protocol: TCP
-    targetPort: 8080
+    targetPort: 8443
```

Comment:
The target port is different: https://github.com/kubernetes-sigs/lws/blob/main/cmd/main.go#L74

Reply:
Can you change the default in main?